console-extensions.json

```diff
@@ -33,6 +44,36 @@
       "component": { "$codeRef": "GatewaySingleOverview" }
     }
   },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "exact": true,
+      "path": "/k8s/ns/:namespace/gateway.networking.k8s.io~v1~HTTPRoute/:name/edit",
+      "component": { "$codeRef": "HTTPRouteCreatePage" }
+    }
+  },
+  {
+    "type": "console.action/resource-provider",
+    "properties": {
+      "model": {
+        "group": "gateway.networking.k8s.io",
+        "version": "v1",
+        "kind": "HTTPRoute"
+      },
+      "provider": { "$codeRef": "useHTTPRouteActions" }
+    }
+  },
+  {
+    "type": "console.resource/create",
```

Comment:
seems to be in here twice? same:

https://github.com/Kuadrant/gateway-api-console-plugin/pull/21/files#diff-2d79046ce9b0f8fda82dd631ce959a36f6c350960d54cf8fdb1579f3c718e83aR22